### PR TITLE
[#349] [BUGFIX]  Affichage des réponses sur QROC (US-401)

### PR DESCRIPTION
**Pull request by MBrandone**

### CODE REVIEW

**Comment by jbuget** [comment]
I've deployed this PR to http://401-bugfix-comparison-window-qroc.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://401-bugfix-comparison-window-qroc.pix.beta.gouv.fr. Please check it out

File: live/app/components/qroc-solution-panel.js [hunk 1]
```diff
@@ -5,6 +5,16 @@ const QrocSolutionPanel = Ember.Component.extend({
   answer: null,
   solution: null,
 
+  isResultOk : Ember.computed('answer', function () {
```
Comment (florianEnoh) [inline live/app/components/qroc-solution-panel.js:8]: Je te propose le refacto suivant : 
  const classMap = {
    isResultOk: 'correction-qroc-box__input-right-answer',
    isResultKo: 'correction-qroc-box__input-wrong-answer',
    isResultWithoutAnswer: 'correction-qroc-box__input-no-answer'
  }

  resultInputClass : Ember.computed('answer', function () {
    return classMap[(this.get('answer.result'))] || '';
  })

File: live/app/templates/components/qroc-solution-panel.hbs [hunk 2]
```diff
@@ -2,14 +2,14 @@
   <div class="rounded-panel__row ">
 
     <div class="correction-qroc-box__answer">
-      <input class="{{if answer.isResultOk "correction-qroc-box__input-right-answer"
-                     (if answer.isResultNotOk "correction-qroc-box__input-wrong-answer"
-                     (if answer.isResultWithoutAnswer "correction-qroc-box__input-no-answer"))}} correction-qroc-box--answer__input"
+      <input class="{{if isResultOk "correction-qroc-box__input-right-answer"
```
Comment (florianEnoh) [inline live/app/templates/components/qroc-solution-panel.hbs:5]: Ceci deviendra donc à binder {{resultInputClass}}

Comment (yojecode) [inline live/app/templates/components/qroc-solution-panel.hbs]: <input class="{{resultInputClass}} correction-qroc-box--answer__input" value="{{answerToDisplay}}" disabled>

**Pull request merged**